.buildkite/pipeline.yml

```
@@ -1,14 +1,61 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+env:
+  SETUP_MAGE_VERSION: '1.14.0'
+  SETUP_GVM_VERSION: 'v0.5.0'
+
 ---
 # $yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 steps:
-  - label: "Build"
-    command: "true"
+  - label: ":buildkite: Lint"
+    command:
```

```suggestion
    command: |
    	mage lint
```

if you use one of the custom images we have

---

Correction, once we use our own set of images this will become 

```bash
echo "--- Mage notice"
mage notice

echo "--- Mage check"
mage -v check

```